test/e2e/launch.js

```diff
@@ -43,12 +44,13 @@ function launch(t) {
       })
 
     started = new Promise(async (resolve, reject) => {
-      console.log("using binary", binary)
+      console.log("using cli binary", binary)
+      console.log("using node binary", nodeBinary)
 
       // TODO cleanup
-      home = newTempDir()
+      cliHome = newTempDir()
```

Comment:
what is `cliHome`?

Reply:
`gaiacli --home XXX`
It stores the config files for the cli and the keys